lib/course_planner/mailer/user_email.ex

```diff
@@ -5,7 +5,9 @@ defmodule CoursePlanner.Mailer.UserEmail do
   use Phoenix.Swoosh, view: CoursePlanner.EmailView, layout: {CoursePlanner.LayoutView, :email}
 
   @notifications %{
-    user_modified: %{subject: "Your profile was updated", template: "user_updated.html"}
+    user_modified: %{subject: "Your profile was updated", template: "user_updated.html"},
```

Comment:
nope, since it's a passive of simple present it should be "is updated".
cause the event happened just now and not somewhere in the past